### PR TITLE
Feat : Add Wishlist Page

### PR DESCRIPTION
**Pull request by OmKakatkar**
- Adds a wishlist page.
- Adds an image `coffee.jpg` inside `assets`.
- `common.css` contains all common styles like navigation.

- [Code Preview](https://codesandbox.io/s/github/OmKakatkar/Ani-Shoppie/tree/feature_wishlist).
- [Live Preview](https://deploy-preview-2--ani-shoppie.netlify.app/static-pages/wishlist/wishlist).

### CODE REVIEW

File: common.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
+:root {
+    --clr-grey-background: #f3f3f3;
+}
+
+.ecom-nav {
+    min-height: 7rem;
+    background-color: var(--white);
+    z-index: 99999;
```
Comment (manish591) [inline common.css:8]: Avoid using large Z-index values.

**Review by manish591 — APPROVED** [review]

File: common.css (hunk 1 above)
Comment (OmKakatkar) [reply common.css:8]: Yes, I wanted the navbar to be on top always, hence the large value. Will change it if a need arises.

Comment (manish591) [reply common.css:8]: Ok

**Review by vinc3nati — APPROVED** [review]

**Comment by vinc3nati** [comment]
Resolve conflicts!

**Pull request merged**